stix_shifter_modules/elastic_ecs/stix_translation/json/to_stix_map.json

```diff
@@ -232,6 +468,816 @@
+      "cybox": false
+    },
+    "syslog": {
+      "facility": {
+        "code": {
+          "key": "x_ecs_log.syslog_facility_code",
+          "cybox": false
+        },
+        "name": {
+          "key": "x_ecs_log.syslog_facility_name",
+          "cybox": false
+        }
+      },
+      "priority": {
+        "key": "x_ecs_log.syslog_priority",
+        "cybox": false
+      },
+      "severity": {
+        "code": {
+          "key": "x_ecs_log.severity_syslog_code",
+          "cybox": false
+        },
+        "name": {
+          "key": "x_ecs_log.severity_syslog_name",
+          "cybox": false
+        }
+      }
+    }
+  },
+  "observer": {
+    "egress": {
+      "zone": {
+        "key": "x_ecs_observer.egress_zone",
+        "cybox": false
+      },
+      "interface": {
+        "alias": {
+          "key": "x_ecs_interface.egress_interface_alias",
+          "cybox": false
+        },
+        "id": {
+          "key": "x_ecs_interface.egress_interface_id",
+          "cybox": false
+        },
+        "name": {
+          "key": "x_ecs_interface.egress_interface_name",
+          "cybox": false
+        }
+      }
+    },
+    "hostname": {
+      "key": "x_ecs_observer.hostname",
+        "cybox": false
+    },
+    "ingress": {
+      "zone": {
+        "key": "x_ecs_observer.ingress_zone",
+        "cybox": false
+      },
+      "interface": {
+        "alias": {
+          "key": "x_ecs_interface.ingress_interface_alias",
+          "cybox": false
+        },
+        "id": {
+          "key": "x_ecs_interface.ingress_interface_id",
+          "cybox": false
+        },
+        "name": {
+          "key": "x_ecs_interface.ingress_interface_name",
+          "cybox": false
+        }
+      }
+    },
+    "ip": {
+      "key": "x_ecs_observer.ip",
+      "cybox": false
+    },
+    "mac": {
+      "key": "x_ecs_observer.mac",
+      "cybox": false
+    },
+    "name": {
+      "key": "x_ecs_observer.name",
+      "cybox": false
+    },
+    "product": {
+      "key": "x_ecs_observer.product",
+      "cybox": false
+    },
+    "serial_number": {
+      "key": "x_ecs_observer.serial_number",
+      "cybox": false
+    },
+    "type": {
+      "key": "x_ecs_observer.type",
+      "cybox": false
+    },
+    "vendor": {
+      "key": "x_ecs_observer.vendor",
+      "cybox": false
+    },
+    "version": {
+      "key": "x_ecs_observer.version",
+      "cybox": false
+    },
+    "os": {
+      "name": {
+        "key": "x_ecs_observer.os_name",
+        "cybox": false
+      },
+      "platform": {
+        "key": "x_ecs_observer.os_platform",
+        "cybox": false
+      },
+      "version": {
+        "key": "x_ecs_observer.os_version",
+        "cybox": false
+      }
+    }
+  },
+  "organization": {
+    "id": {
+      "key": "x_ecs_organization.id",
+      "cybox": false
+    },
+    "name": {
+      "key": "x_ecs_organization.name",
+      "cybox": false
+    }
+  },
+  "pe": {
```

Comment:
It looks like `pe` gets embedded under dll.pe, file.pe, process.pe

Reply:
right. didn't notice the last part of the reference doc.